engine/launcher/src/test/scala/org/enso/launcher/NativeTest.scala

```diff
@@ -117,6 +115,7 @@ trait NativeTest extends AnyWordSpec with Matchers with TimeLimitedTests {
     if (!Files.isExecutable(path)) {
       throw new RuntimeException("Failed to make it executable...")
     }
+    Thread.sleep(100)
```

Comment:
Why?

Reply:
I should explain it better indeed.

It is very rare, but every once in a while the tests that use the copied launcher fail (at least on Linux) with an error along the lines of "device/file is busy". I believe it is because I try to call the launcher immediately after copying the file and the filesystem does not like it, so I thought adding a very small time-margin would resolve the issue. I cannot say for sure, because even without it the failure would happen very rarely, so I cannot tell if this helped it or I just was (un)lucky.

Reply:
I added a [comment](https://github.com/enso-org/enso/pull/1059/commits/698c5d30eab678b2618d58eae9fae8201d760e5d#diff-9cf1ffddd9dc670020249c7a3ec38eb2R108-R125) explaining the situation. Do you think this is reasonable?

Reply:
Works for me.